### PR TITLE
team/tidb: a vote for fixdb as the reviewer

### PR DESCRIPTION
**Pull request by winoros**
as the title said

### CODE REVIEW

**Comment by winoros** [comment]
(And a name alignment change.

**Review by bb7133 — APPROVED** [review]
+1

File: votes/0700-fixdb-as-tidb-reviewer.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# A Vote for fixdb as TiDB reviewer 
+
+## Proposal
+
+[@fixdb](https://github.com/fixdb) has actively contributed to TiDB's planner and have a deep knowledge of the optimizer module of the morden datebase system.
```
Comment (fixdb) [inline votes/0700-fixdb-as-tidb-reviewer.md:5]: morden datebase -> modern database?

Comment (fixdb) [inline votes/0700-fixdb-as-tidb-reviewer.md:5]: have a deep knowledge -> has a deep knowledge

File: teams/tidb/membership.json [hunk 2]
```diff
@@ -90,6 +90,7 @@
     "reviewers": [
         "ChenPeng2013",
         "Defined2014",
+	"fixdb",
```
Comment (fixdb) [inline teams/tidb/membership.json:93]: more spaces to align with the previous lines?

Comment (winoros) [reply teams/tidb/membership.json:93]: Oh, i used the tab wrongly

File: votes/0700-fixdb-as-tidb-reviewer.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
+# A Vote for fixdb as TiDB reviewer 
+
+## Proposal
+
+[@fixdb](https://github.com/fixdb) has actively contributed to TiDB's planner and has a deep knowledge of the optimizer module of the modern datebase system.
```
Comment (fixdb) [inline votes/0700-fixdb-as-tidb-reviewer.md:5]: datebase -> database

**Comment by winoros** [comment]
Since there are two +1 and no -1. We merged this pull.

**Pull request merged**